stable/fluentd/templates/deployment.yaml

```diff
@@ -89,6 +89,7 @@ spec:
           mountPath: /etc/fluent/config.d
         - name: buffer
           mountPath: "/var/log/fluentd-buffers"
+      serviceAccountName: {{ .Values.serviceAccountName }}
```

Comment:
isn't the default value `{{ template "fluentd.fullname" . }}`  today ? Why should we set it as default 🤔  ?

Reply:
Maybe I am missing something but I noticed that my pod is running under the default service account in my namespace (which is default) and I don't want to change the namespace. Are you telling me that if I create a service account named using the fullname It will use it instead?

Reply:
Yeah that worked! thanks!

Reply:
I was not able to get my pod to use the service account I created. Without modifying the service account name to use the fullname.